Storage/tests/Unit/BucketTest.php

```diff
@@ -118,6 +118,47 @@ public function testUploadData()
         );
     }
 
+    public function testUploadFolderData()
+    {
+        $i=0;
+        while(file_exists( $dir = sys_get_temp_dir().DIRECTORY_SEPARATOR.'google'.$i )){
+            $i++;
+        }
```

Comment:
I'd make the format consistent with the rest of the code (e.g. lines 134-143 here, or 398-407 of `Bucket.php`). That is, instead of
```
        while(file_exists( $dir = sys_get_temp_dir().DIRECTORY_SEPARATOR.'google'.$i )){
            $i++;
        }
```
I'd rather have
```
        while (file_exists($dir = sys_get_temp_dir().DIRECTORY_SEPARATOR.'google'.$i))
        {
            $i++;
        }
```